articles/active-directory/develop/quickstarts/active-directory-android.md

```diff
@@ -0,0 +1,180 @@
+---
+title: Azure AD v2 Android quickstart | Microsoft Docs
+description: Learn how Android native applications can call an API that requires access tokens by Azure Active Directory v2.0 endpoint.
+services: active-directory
+documentationcenter: dev-center-name
+author: andretms
+manager: mtillman
+editor: ''
+
+ms.assetid: 820acdb7-d316-4c3b-8de9-79df48ba3b06
+ms.service: active-directory
+ms.component: develop
+ms.devlang: na
+ms.topic: article
+ms.tgt_pltfrm: na
+ms.workload: identity
+ms.date: 07/20/2018
+ms.author: andret
+ms.custom: aaddev 
+#Customer intent: As an application developer, I want to learn how Android native apps can call an API that requires access tokens by Azure AD v2.0 endpoint.
+---
+
+# Quickstart: Sign in users and call the Microsoft Graph API from an Android app
+
+This quickstart contains a code sample that demonstrates how a native Android application can sign in personal, work and school accounts, get an access token, and call the Microsoft Graph API.
+
+![How the sample app generated by this Quickstart works](media/active-directory-android/android-intro.png)
+
+> [!NOTE]
+> **Prerequisites**
+> * Android Studio
+> * Android SDK 21 or later is required (SDK 25 is recommended)
+> * An Android device with Google Chrome or a web browser with support for Custom Tabs is required for this release of MSAL for Android
+
+> [!div renderon="docs"]
+> ## Register your application and download your quickstart app
+> You have two options to start your quickstart application:
+> * [Express] [Option 1: Register and auto configure your app and then download your code sample](#option-1-register-and-auto-configure-your-app-and-then-download-your-code-sample)
+> * [Manual] [Option 2: Register and manually configure your application and code sample](#option-2-register-and-manually-configure-your-application-and-code-sample)
+>
+> ### Option 1: Register and auto configure your app and then download your code sample
+>
+> 1. Go to the [Azure portal - Application Registration](https://portal.azure.com/signin/index/?Microsoft_AAD_RegisteredApps=true#blade/Microsoft_AAD_RegisteredApps/applicationsListBlade/quickStartType/androidQuickstartPage/sourceType/docs).
+> 1. Enter a name for your application and click **Register**.
+> 1. Follow the instructions to download and automatically configure your new application for you in one click.
+> [!div renderon="docs"]
+> ### Option 2: Register and manually configure your application and code sample
+> #### Step 1: Register your application
+> 
+> 1. To register an application, go to the [Azure portal - Application Registration [Prod]](https://aka.ms/registeredappsprod) and select **New registration**.
+> 1. Enter a name for your application and click **Register**.
+> 1. Select the **Authentication** page, then add `msal{AppId}://auth` (where *{AppId}* is the application ID from the application you just registered), select **Installed client** under **Type**, and then select **Save**.
```

Comment:
I've asked this a few times, but I forgot. Is this not auto-registered for new apps?